fondant/component_spec.py

```diff
@@ -140,40 +140,34 @@ def image(self):
         return self._specification["image"]
 
     @property
-    def input_subsets(self) -> t.Mapping[str, ComponentSubset]:
+    def consumes(self) -> t.Mapping[str, ComponentSubset]:
         """The input subsets of the component as an immutable mapping."""
```

Comment:
Do we then still describe the `consumed subsets` as `input subsets`? or is input subsets just indicative of the additional subsets defined with the `additionalSubsets` flag. If we're going for this notation switch would rather remove the term input subsets completely to avoid confusion